blocks/header/header.js

```diff
@@ -1,7 +1,23 @@
-import { loadCSS, loadScript } from '../../scripts/aem.js';
+import { loadCSS, loadScript, getMetadata } from '../../scripts/aem.js';
 import ffetch from '../../scripts/ffetch.js';
 import { link } from '../../scripts/dom-helpers.js';
 
+/**
+  * Use getMetadata to get the locale of the page
+  * * Update the html lang attribute to the locale
+  * If the language is Arabic, Hebrew or Kuwaiti Arabic, set the direction to rtl
+  */
+async function setLocaleAndDirection() {
+  const locale = getMetadata('og:locale') || 'en-us';
+  const dir = (locale === 'ar-sa' || locale === 'he-il' || locale === 'ar-kw') ? 'rtl' : 'ltr';
+  document.querySelector('html').setAttribute('dir', dir);
+
+  const lang = (locale === 'en-us') ? 'en' : locale;
+  document.querySelector('html').setAttribute('lang', lang);
+}
+
+await setLocaleAndDirection();
```

Comment:
Can you move it to the "decorate" function? Code is easier to maintain if there's a single entrypoint (the exported default function)